openvinotoolkit/datumaro
# Support mask annotations for CVAT data format

### Summary

- Resolve #1074 
- Support mask annotation import/export for CVAT data format

### How to test
I update save and load test for this change and mannually check it for the CVAT exported dataset with mask annotations: [car_tree_bug_seg_cvat_project.zip](https://github.com/openvinotoolkit/datumaro/files/11956199/car_tree_bug_seg_cvat_project.zip).
![image](https://github.com/openvinotoolkit/datumaro/assets/26541465/fb8883e0-2a94-44c0-be0f-cab3bdfafd83)


### Checklist


### License

  Feel free to contact the maintainers if that's a concern.

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```
